src/citrine/informatics/executions/generative_design_execution.py

```diff
@@ -0,0 +1,111 @@
+    _session: Optional[Session] = None
+    project_id: Optional[UUID] = None
+
+    uid: UUID = properties.UUID('id', serializable=False)
+    """:UUID: Unique identifier of the workflow execution"""
+    status = properties.Optional(properties.String(), 'status', serializable=False)
+    """:Optional[str]: short description of the execution's status"""
+    status_description = properties.Optional(
+        properties.String(), 'status_description', serializable=False)
+    """:Optional[str]: more detailed description of the execution's status"""
+    status_info = properties.Optional(
+        properties.List(properties.String()),
+        'status_info',
+        serializable=False
+    )
+    """:Optional[List[str]]: human-readable explanations of the status"""
+    created_by = properties.Optional(properties.UUID, 'created_by', serializable=False)
+    """:Optional[UUID]: id of the user who created the resource"""
+    updated_by = properties.Optional(properties.UUID, 'updated_by', serializable=False)
+    """:Optional[UUID]: id of the user who most recently updated the resource,
+    if it has been updated"""
+    create_time = properties.Optional(properties.Datetime, 'create_time', serializable=False)
+    """:Optional[datetime]: date and time at which the resource was created"""
+    update_time = properties.Optional(properties.Datetime, 'update_time', serializable=False)
+    """:Optional[datetime]: date and time at which the resource was most recently updated,
+    if it has been updated"""
+
+    def __init__(self):
+        """GenerativeDesign executions are not directly instantiated by the user."""
+        pass  # pragma: no cover
+
+    def __str__(self):
+        return '<GenerativeDesignExecution {!r}>'.format(str(self.uid))
+
+    def _path(self):
+        return format_escaped_url(
+            # TODO: Check this path once the backend is ready
+            '/projects/{project_id}/generative-design/',
+            project_id=self.project_id,
+        )
+
+    @classmethod
+    def _build_results(
+        cls, subset_collection: Iterable[dict]
+    ) -> Iterable[GenerationExecutionResult]:
+        for generation_result in subset_collection:
+            yield GenerationExecutionResult.build(generation_result)
+
+    def results(
+        self,
+        *,
+        page: Optional[int] = None,
+        per_page: int = 100,
+    ) -> Iterable[GenerationExecutionResult]:
+        """Fetch the Design Candidates for the particular execution, paginated."""
+        # TODO: Check this path once the backend is ready
+        path = self._path() + '/results'
+
+        fetcher = partial(self._fetch_page, path=path, fetch_func=self._session.get_resource)
+
+        return self._paginator.paginate(page_fetcher=fetcher,
+                                        collection_builder=self._build_results,
+                                        page=page,
+                                        per_page=per_page)
+
+
+    # def result(
```

Comment:
Combining what I see here with what you said in slack, I think you're trying to get a single result, but from an endpoint that isn't just `_path_template`.

You'll need to call `session.get_resource` directly. See the example in [report.py](https://github.com/CitrineInformatics/citrine-python/blob/32403f6139f1787223dc9fe7effd66d62f72a1a0/src/citrine/resources/report.py#L45).

Reply:
Similar to above, this `result` method should be sure to only return a result if the given ID is associated with the execution in which it's contained.